sdk-rs/src/store.rs

```diff
@@ -4,33 +4,25 @@ use crate::{error::Error, Client as SdkClient};
 use base64::{engine::general_purpose, Engine as _};
 use reqwest::header::{HeaderValue, AUTHORIZATION};
 use serde::{Deserialize, Serialize};
+use serde_with::{base64::Base64, serde_as};
 
 /// The JSON payload for a `get` operation response.
+#[serde_as]
 #[derive(Serialize, Deserialize, Debug)]
 pub struct GetResultPayload {
-    pub value: String,
-}
-
-/// The result of a `get` operation.
-#[derive(Debug)]
-pub struct GetResult {
-    /// The retrieved value.
+    #[serde_as(as = "Base64")]
     pub value: Vec<u8>,
 }
 
-/// An item in the result of a `query` operation. For internal use.
+/// An item in the result of a `query` operation.
+#[serde_as]
 #[derive(Serialize, Deserialize, Debug)]
 pub struct QueryResultItemPayload {
-    pub key: String,
-    pub value: String,
-}
-
-/// An item in the result of a `query` operation.
-#[derive(Debug)]
-pub struct QueryResultItem {
     /// The key of the item.
-    pub key: String,
+    #[serde_as(as = "Base64")]
```

Comment:
I suppose this is probably better than implementing our own codec for JSON 👁️

Reply:
I decided to use it when I noticed some of our Vec<u8> getting silently encoded as integer vectors. This makes it harder to screw up I think.